### PR TITLE
Pivot, AutoSuggestBox, TextBox, XamlReader updates

### PR DESCRIPTION
**Pull request by jeromelaban**
- Added missing `TransformGroup` `ContentProperty`
- Fixed invalid namespace attribution of attached properties in `XamlReader`
- Fixed `BitmapImage.UriSource` udpates not being applied on Wasm
- Add basic implementation of `AutoSuggestBox`
- Fixed focus stealing issues with inactive `PivotItem` content
- Add `ThreadPoolTimer` support
- Fix for iOS popup not appearing
- Fix for Wasm textbox not properly updating while not loaded

### CODE REVIEW

File: src/Uno.UI/UI/Xaml/Controls/Pivot/Pivot.cs [hunk 1]
```diff
@@ -190,10 +190,12 @@ private void SynchronizeSelectedItem()
 					{
 						pivotItem.Opacity = 0;
 						pivotItem.IsHitTestVisible = false;
+						pivotItem.Visibility = Visibility.Collapsed;
```
Comment (Dissolute) [inline src/Uno.UI/UI/Xaml/Controls/Pivot/Pivot.cs]: Does it still need to set Opacity and IsHitTestVisible if it's setting visibility to Collapsed?

File: src/Uno.UI/Generated/3.0.0.0/Windows.UI.Xaml.Controls/AutoSuggestBox.cs [hunk 2]
```diff
@@ -7,7 +7,7 @@ namespace Windows.UI.Xaml.Controls
 	#endif
 	public  partial class AutoSuggestBox : global::Windows.UI.Xaml.Controls.ItemsControl
 	{
-		#if __ANDROID__ || __IOS__ || NET46 || __WASM__
+#if false
```
Comment (carldebilly) [inline src/Uno.UI/Generated/3.0.0.0/Windows.UI.Xaml.Controls/AutoSuggestBox.cs]: Why not just remove it ? ;-)

File: src/Uno.UI/UI/Xaml/Controls/AutoSuggestBox/AutoSuggestBox.cs [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+			if (IsLoaded)
+			{
+				RegisterEvents();
+			}
+		}
+
+		void RegisterEvents()
+		{
+			_textBox.KeyDown += OnTextBoxKeyDown;
```
Comment (carldebilly) [inline src/Uno.UI/UI/Xaml/Controls/AutoSuggestBox/AutoSuggestBox.cs]: No null check if template badly designed.

File: src/Uno.UI/UI/Xaml/Controls/AutoSuggestBox/AutoSuggestBox.cs [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+			{
+				SubmitSearch();
+			}
+		}
+
+		private static void OnTextChanged(DependencyObject dependencyObject, DependencyPropertyChangedEventArgs args)
+		{
+			if(dependencyObject is AutoSuggestBox tb)
+			{
+				tb.TextChanged.Invoke(tb, new AutoSuggestBoxTextChangedEventArgs() { Reason = AutoSuggestionBoxTextChangeReason.UserInput });
```
[the remark this answers is not in the document]
Comment (jeromelaban) [reply src/Uno.UI/UI/Xaml/Controls/AutoSuggestBox/AutoSuggestBox.cs]: Definitely can :)

[the remark this answers is not in the document]
Comment (weitzhandler) [reply src/Uno.UI/UI/Xaml/Controls/AutoSuggestBox/AutoSuggestBox.cs]: Should be `tb.TextChanged?.Invoke` then.

File: src/Uno.UI/UI/Xaml/Controls/Pivot/Pivot.cs (hunk 1 above)
Comment (weitzhandler) [reply src/Uno.UI/UI/Xaml/Controls/Pivot/Pivot.cs]: 1. Same [here](https://github.com/nventive/Uno/pull/77/files#diff-b075b6b5f6cb0c0eb8643ddfd5035f1bR197)
2. I'm not so involved, but setting the opacity and hit-test-visibility here might take away custom configuration (e.g. lower opacity) from the user and will reset it on selection synchronization.

File: src/Uno.UI/Generated/3.0.0.0/Windows.UI.Xaml.Controls/AutoSuggestBox.cs (hunk 2 above)
Comment (jeromelaban) [reply src/Uno.UI/Generated/3.0.0.0/Windows.UI.Xaml.Controls/AutoSuggestBox.cs]: This is for easier change tracking when the source generator is executed.

File: src/Uno.UI/UI/Xaml/Controls/Pivot/Pivot.cs (hunk 1 above)
Comment (jeromelaban) [reply src/Uno.UI/UI/Xaml/Controls/Pivot/Pivot.cs]: Fair enough, this is an extrapolation of the behavior coming from UWP, where the opacity seems to be changed as well, but this is not required for the current implementation.

Collapsed-only will be enough.

**Pull request merged**